### ISSUE TITLE
Problem serializing amm_swaps

### ISSUE THREAD

**Issue by yabirgb** [rotki/rotki#4682]
## Problem Definition

In the PR https://github.com/rotki/rotki/pull/4534 we changed the `tx_hash` from being a str to a blob. This field is used as link https://github.com/rotki/rotki/blob/21f5673e818f701311649bf873a452e8a1f657c9/rotkehlchen/db/schema.py#L794 and this causes a serialization error by the `TradeSchema` https://github.com/rotki/rotki/blob/21f5673e818f701311649bf873a452e8a1f657c9/rotkehlchen/api/v1/schemas.py#L656 when doing

```
trade_schema.dump(trade)
```
at https://github.com/rotki/rotki/blob/21f5673e818f701311649bf873a452e8a1f657c9/rotkehlchen/api/rest.py#L730 

## Logs

```
    serialized_trade = self.trade_schema.dump(trade)
  File "/home/yabirgb/work/rotki/.venv/lib/python3.9/site-packages/marshmallow/schema.py", line 557, in dump
    result = self._serialize(processed_obj, many=many)
  File "/home/yabirgb/work/rotki/.venv/lib/python3.9/site-packages/marshmallow/schema.py", line 525, in _serialize
    value = field_obj.serialize(attr_name, obj, accessor=self.get_attribute)
  File "/home/yabirgb/work/rotki/.venv/lib/python3.9/site-packages/marshmallow/fields.py", line 342, in serialize
    return self._serialize(value, attr, obj, **kwargs)
  File "/home/yabirgb/work/rotki/.venv/lib/python3.9/site-packages/marshmallow/fields.py", line 893, in _serialize
    return utils.ensure_text_type(value)
  File "/home/yabirgb/work/rotki/.venv/lib/python3.9/site-packages/marshmallow/utils.py", line 211, in ensure_text_type
    val = val.decode("utf-8")
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe3 in position 0: invalid continuation byte
[10/08/2022 19:18:12 CEST] CRITICAL rotkehlchen.api.server Greenlet-6: Unhandled exception when processing endpoint request exc_info=True, exception='utf-8' codec can't decode byte 0xe3 in position 0: invalid continuation byte
```

### System Description

Operating system: Linux
Rotki version: 1.25.1

**Comment by LefterisJP** [comment]
Weird. How come nobody has reported this?

We want to get rid of this ammswaps DB table monstrosity anyway so perhaps consider that to be the fix?

**Comment by LefterisJP** [comment]
Should be fixed by #4693
